test/cases/ostree-vsphere.sh

```diff
@@ -0,0 +1,884 @@
+#!/bin/bash
+
+# This test scrip covers:
+# 1. build edge-container image, build edge-vsphere image
+# 2. upload image to vsphere datacenter7.0-intel, create edge vm, run test cases against vm.
+# 3. upload image to vsphere datacenter7.0-amd, create edge vm, run test cases against vm.
+# 4. build upgrade edge-container image
+# 5. run rpm-ostree upgrade in all edge vms, then run test cases against upgraded vms.
+# 6. delete all vsphere vms, images
+
+set -euo pipefail
+
+# Provision the software under test.
+/usr/libexec/osbuild-composer-test/provision.sh none
+
+# Get OS data.
+source /etc/os-release
+ARCH=$(uname -m)
+
+source /usr/libexec/tests/osbuild-composer/shared_lib.sh
+
+# Install sshpass
+sudo dnf install -y sshpass
+
+# Install ansible module
+sudo ansible-galaxy collection install community.vmware
+sudo pip3 install PyVmomi
+
+# Install govc
+sudo curl -L -o - "https://github.com/vmware/govmomi/releases/latest/download/govc_$(uname -s)_$(uname -m).tar.gz" | tar -C /usr/local/bin -xvzf - govc
```

Comment:
Try to avoid `latest` version. Let's make this a fixed version. https://github.com/vmware/govmomi/releases/download/v0.30.5/govc_Linux_x86_64.tar.gz

Reply:
Fixed, thanks.